mmv1/products/accesscontextmanager/api.yaml

```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://accesscontextmanager.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://accesscontextmanager.googleapis.com/v1/
```

Comment:
Although the new properties are available in the v1 API version, the documentation indicates they are not GA (see the [docs](https://cloud.google.com/vpc-service-controls/docs/ingress-egress-rules)), hence why the same base_url is used for both ga and beta versions.